src/raw-transaction-status.ts

```diff
@@ -1,6 +1,43 @@
+import { TransactionStatus } from './generated/web/legacy/transaction_status_pb'
+import { GetTxResponse } from './generated/web/rpc/v1/tx_pb'
+
 /** Abstraction around raw Transaction Status for compatibility. */
-export default interface RawTransactionStatus {
-  getValidated(): boolean
-  getTransactionStatusCode(): string
-  getLastLedgerSequence(): number
+export default class RawTransactionStatus {
+  /**
+   * Create a RawTransactionStatus from a TransactionStatus legacy protocol buffer.
+   */
+  public static fromTransactionStatus(
```

Comment:
Everything in a TypeScript class is `public` by default.

Do you find it valuable to explicitly state that things are `public`? (If you do, I wonder if we should enforce that in ESLint across our projects somehow).

Reply:
I tend to value being explicit and in other languages you don't get public by default. I'll roll these changes back.